courses/serializers.py

```diff
@@ -536,3 +536,28 @@ class Meta:
             "certificate",
             "receipt",
         ]
+
+
+class CourseTopicSerializer(serializers.ModelSerializer):
+    """
+    CourseTopic model serializer
+    """
+
+    course_count = serializers.SerializerMethodField()
+
+    class Meta:
+        model = models.CourseTopic
+        fields = ["name", "parent", "course_count"]
```

Comment:
A thought, you can take it or leave it. We don't need the `parent` key in the response or the`CourseTopicViewSet ` since we only use it to respond with parent topics, `parent` key in the API response is always going to be `null`.